tethysapp/temp_precip_trends/app.py

```diff
@@ -18,19 +18,8 @@ class TempPrecipTrends(TethysAppBase):
     enable_feedback = False
     feedback_emails = []
 
-    THREDDS_SERVICE_NAME = 'thredds_service'
-
-    def spatial_dataset_service_settings(self):
-        sds_settings = (
-            SpatialDatasetServiceSetting(
-                name=self.THREDDS_SERVICE_NAME,
-                description='THREDDS service for app to use',
-                engine=SpatialDatasetServiceSetting.THREDDS,
-                required=True,
-            ),
-        )
-
-        return sds_settings
+    SET_THREDDS_SDS_NAME = 'primary_thredds'
+    SET_THREDDS_DATASET_NAME = 'ERA5 Daily Precipitation and Temperatures'
```

Comment:
Sorry, I wasn't clear enough on this. Create a `CustomSetting` with a default value of `'ERA5 Daily Precipitation and Temperatures'`. That way we don't have to change code if the dataset name changes, just change the setting. The `SET_THREDDS_DATASET_NAME` constant should be the name of the `CustomSetting`.